sphinx_gherkindoc/cli.py

```diff
@@ -18,6 +19,19 @@
 DEFAULT_TOC_DEPTH = 4
 
 
+def _get_function_from_command_line_arg(module_func_str: str) -> Callable:
```

Comment:
NB: Huh, just noticed that this should probably be named `_get_function_from_string_doodady_thing` since it doesn't really matter where said string actually originated.
But this is an internal function and a better name for "doodady_thing" is not coming instantly to mind, so it's an RR for when inspiration strikes.